virtcontainers/network_test.go

```diff
@@ -599,3 +688,106 @@ func TestGenerateInterfacesAndRoutes(t *testing.T) {
 		"Routes returned didn't match: got %+v, expecting %+v", resRoutes, expectedRoutes)
 
 }
+
+func TestGenerateRandomPrivateMacAdd(t *testing.T) {
+	assert := assert.New(t)
+
+	addr, err := generateRandomPrivateMacAddr()
```

Comment:
Thanks for adding. Could you also make a second call to `generateRandomPrivateMacAddr()` so that we can check to ensure the results are different?

Reply:
@jodh-intel Done. I have addressed all of @sboeuf's comments. For those pending we had a discussion about it offline. I have added comments for reference.